src/components/flag/flag.scss

```diff
@@ -0,0 +1,20 @@
+.flag-img {
+  object-fit: cover;
+  img {

```

Comment:
This style is not used, since .flag-img is already an image, so it doesn't contain img tag inside.